### PR TITLE
TE-16.3: Updated script to program with non-existing tunnel destination as per README.

### PR DESCRIPTION
**Pull request by mohanasm**
Added below code changes for TE-16.3 
* For Testcase-8 and Testcase-9, expectation is to program with the tunnel destination for which route is non-existing. Current script uses the route which is existing in the table. Updated the code with the IP address specified in the README which is non-existing in the specified table. 
* Removed explicit disable of auto-negotiation and setting port-speed,duplex for specific PMD type. It is not a requirement per README.

### CODE REVIEW

**Comment by rohit-rp** [comment]
Please rebase and submit again. I will review asap

**Comment by rohit-rp** [comment]
Please rebase and submit again, I will review asap

**Comment by mohanasm** [comment]
> Please rebase and submit again. I will review asap

Done. Please check.

File: feature/gribi/otg_tests/encap_frr_with_reencap_vrf_test/encap_frr_with_reencap_vrf_test.go [hunk 1]
```diff
@@ -249,13 +253,6 @@ func dutInterface(p *ondatra.Port, dut *ondatra.DUTDevice) *oc.Interface {
 		i.Enabled = ygot.Bool(true)
 	}
 
-	if p.PMD() == ondatra.PMD100GBASEFR {
```
Comment (rohit-rp) [inline feature/gribi/otg_tests/encap_frr_with_reencap_vrf_test/encap_frr_with_reencap_vrf_test.go:252]: removing this is causing failures for Arista, if it is not needed, please put it in a deviation for Juniper

**Review by rohit-rp — CHANGES_REQUESTED** [review]

File: feature/gribi/otg_tests/encap_frr_with_reencap_vrf_test/encap_frr_with_reencap_vrf_test.go (hunk 1 above)
Comment (mohanasm) [reply feature/gribi/otg_tests/encap_frr_with_reencap_vrf_test/encap_frr_with_reencap_vrf_test.go:252]: Moved the code for explicit configuration of port speed under the deviation. Please check.

**Review by Swetha-haridasula — APPROVED** [review]
LGTM

**Pull request merged**